### PR TITLE
T27545 (Debian)

### PR DESCRIPTION
**Pull request by GeorgesStavracas**
**Do not merge yet**

Dropped the workarounds to test failures, and split the `eos-metrics` dependency into a separate commit.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -13,6 +13,7 @@ Build-Depends: debhelper-compat (= 12),
                at-spi2-core <!nocheck>,
                dbus <!nocheck>,
                dmz-cursor-theme <!nocheck>,
+               eos-metrics-0-dev,
```
Comment (andrunko) [inline debian/control]: Please mention this in the changelog as well.

File: debian/changelog [hunk 2]
```diff
@@ -1,8 +1,278 @@
-mutter (3.32.1-1endless0) eos; urgency=medium
```
Comment (andrunko) [inline debian/changelog]: On gsd rebase I've kept the old endless entries for reference, not sure if worth doing the same here.

Comment (GeorgesStavracas) [reply debian/changelog]: We don't usually leave the previous changelog entries when rebases.

Comment (andrunko) [reply debian/changelog]: Ack, I've updated https://github.com/endlessm/gnome-settings-daemon/pull/59 to remove the old entries as well.

**Pull request closed without merging**